### ISSUE TITLE
Pop!OS compatibility

### ISSUE THREAD

**Issue by Draco18s** [B00merang-Project/Windows-10#205]
The "legacy applications" appearance setting causes text (and some buttons) in the PopShop application to be the same color as the background.

![image](https://user-images.githubusercontent.com/1750680/165803051-bbcc4d80-ba77-41bd-927c-ebaa1f6a67a2.png)

What it looks like normally:

![image](https://user-images.githubusercontent.com/1750680/165803258-0ac93bac-8ebc-49c7-bb5a-e1ba057be559.png)

**Comment by Elbullazul** [comment]
I'll see what I can do

**Comment by Draco18s** [comment]
Yeah, its one of those weird things that's (to me) "how the heck would that do that?"
But I have no idea how the theme system works, under the hood (I've dug into the files, and while I understand what I'm looking at, I have no idea how to identify the part I need to change to affect a desired outcome).

**Comment by Draco18s** [comment]
> is there something special needed to change themes in pop! os? I installed gnome tweaks but the themes don't show up

I do know that they are installed at `/usr/share/themes`, so if you download the repo as a zip file and extract it into that directory (contained in a folder, forget if the zip has a root folder or not) it should show up in Tweaks just fine.
![image](https://user-images.githubusercontent.com/1750680/166523980-b30359d6-9cc1-4a63-888f-29e5ab87b95e.png)
